src/components/ha-picture-upload.ts

```diff
@@ -46,6 +47,9 @@ export class HaPictureUpload extends LitElement {
 
   @property({ type: Number }) public size = 512;
 
+  @property({ type: Number, attribute: false })
+  public opacityPercentage?: number;
```

Comment:
Can we use a css variable instead of a property : `--picture-opacity` ?

Reply:
Yeah that sounds good. I'll work on it :+1: